website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -775,6 +817,94 @@ spec:
 ```
 {{% /alert %}}
 
+### AL2023
+
+* Your UserData may be in one of three formats: a [MIME multi part archive](https://cloudinit.readthedocs.io/en/latest/topics/format.html#mime-multi-part-archive), a NodeConfig YAML / JSON string, or a shell script.
+* Karpenter will transform your custom UserData as a MIME part, if necessary, and then merge a generated MIME part to the **beginning** of the the generated MIME multi part archive. Since `nodeadm`, the bootstrap for the AL2023 EKS optimized AMI, merges NodeConfig objects from top to bottom any fields specified in the custom UserData will override Karpenter generated fields. Take caution when overriding these fields, misconfiguration can result in Nodes being unable to connect to your cluster.
```

Comment:
I think this first sentence is a little confusing. Is there a way to bring a bit more clarity to what you are meaning here.

Reply:
Agreed, could definitely be better. Thoughts on this?
> Karpenter will transform your custom UserData into a MIME part, if necessary, and then create a MIME multi-part archive. This archive will consist of a generated NodeConfig, containing Karpenter's default values, followed by the custom UserData. For more information on how this UserData is handled by the AMI, refer to (link to nodeadm docs).